packages/node_modules/@webex/plugin-meetings/internal-README.md

```diff
@@ -0,0 +1,145 @@
+# Contributing Code
+
+> Similar to the [standard contributing insructions](https://github.com/webex/spark-js-sdk/blob/master/CONTRIBUTING.md#contributing-code).
+
+## Build Dependencies
+
+Before you can build the Cisco Webex JS SDK, you will need:
+
+- [Node.js](https://nodejs.org/) (LTS)
+  - We recommend using [nvm](https://github.com/creationix/nvm) (or [nvm-windows](https://github.com/coreybutler/nvm-windows))
+    to easily switch between Node.js versions.
+  - Install the latest Node.js Long Term Support using `nvm install --lts`
+  - Install the latest npm to enable security audits using `npm install npm@latest -g`
+- [Git](https://git-scm.com/)
+
+### .env
+
+Create a file called `.env` that defines:
+
+CISCOSPARK_CLIENT_SECRET="[get the shared client secret here](https://cisco.box.com/s/phyd6usx1ga6vf06tdzx5bxn9epnwc2w)"
+
+### npm
+
+- [Enable 2-factor authentication](https://docs.npmjs.com/getting-started/using-two-factor-authentication)
+- [Sign into NPM](https://docs.npmjs.com/private-modules/ci-server-config#how-to-create-a-new-authentication-token) so that you can install the private package `@ciscospark/test-helper-test-users`. This package generates temporary test users.
+
+### SauceLabs
+
+[Follow these instructions](https://github.com/webex/spark-js-sdk/blob/master/SCRIPTS.md#saucelabs) to use SauceLabs.
+
+## Build the SDK
+
+Fork the [spark-js-sdk](https://github.com/webex/spark-js-sdk/) repository and `git clone` your fork:
+
+```bash
+git clone https://github.com/YOUR-USERNAME/spark-js-sdk.git
+```
+
+Install tooling dependencies with:
+
+```bash
```

Comment:
We should comment about `nvm` and installing the correct version of `npm` here.

Reply:
@adamweeks I have instructions about that [at the top](https://github.com/webex/spark-js-sdk/pull/997/files/887f100a9ea11225c544f2ace8c956da10e736e5#diff-5541a3cf2b97b0667fb71e0b3212e370R10). Should I put a note to refer to that above?

Reply:
Yea, that would be good